src/Makefile

```diff
@@ -34,13 +40,14 @@ DEFINES+=-DREMUSER=\"${OSSEC_USER_REM}\"
 DEFINES+=-DGROUPGLOBAL=\"${OSSEC_GROUP}\"
 DEFINES+=-DMAILUSER=\"${OSSEC_USER_MAIL}\"
 
-LDFLAGS+=-lm
+OSSEC_LDFLAGS=${LDFLAGS} -lm
```

Comment:
Agreed - much much better - kicking myself for ever writing the way it was before :)